src/services/expense.service.js

```diff
@@ -0,0 +1,66 @@
+const { v4: uuidv4 } = require('uuid');
+
+let expenses = [];
+
+const resetExpenses = () => {
+  expenses = [];
+};
+
+const getAll = () => {
+  return expenses;
+};
+
+const getById = (id) => {
+  return expenses.find((expense) => expense.id === id) || null;
+};
+
+const create = ({ userId, title, amount, category, note }) => {
+  const newExpense = {
+    id: uuidv4(),
+    userId,
+    spentAt: new Date(),
+    title,
+    amount,
+    category,
+    note,
+  };
+
+  expenses.push(newExpense);
+
+  return newExpense;
+};
+
+const remove = (id) => {
+  const filteredExpenses = expenses.filter((item) => item.id !== id);
+
+  expenses = filteredExpenses;
+
+  return expenses;
```

Comment:
The `remove` function returns the entire `expenses` array, which might not be necessary. Consider returning a boolean indicating success or failure instead.